mobile/src/employee-details/employee-details-events-list.tsx

```diff
@@ -198,11 +194,11 @@ class EmployeeDetailsEventsListImpl extends Component<EmployeeDetailsEventsListP
         }
 
         switch (event.status) {
-            case CalendarEventStatus.Requested:
-            case CalendarEventStatus.Approved:
-                return CalendarEventStatus.Requested;
-            case CalendarEventStatus.Processed:
-                return CalendarEventStatus.Approved;
```

Comment:
It was there before so I don't think it's a good idea to change this in the same PR. But this code is, well... surprising.